### DISCUSSION TITLE
Replace new line "\r\n"

### DISCUSSION THREAD

**Discussion by ramisedhom** [hluk/CopyQ#2083] [Q&A] [answered]
I would like to replace new line "\r\n" (Windows EOL)

I write this script:
![image](https://user-images.githubusercontent.com/1517228/189496419-c717b5d6-b919-4d5e-af33-0db89eae38f8.png)

but when script is saved, it escapes the \r and \n as below:
```
[Command]
Name=Add extra line in selection
Command="
    copyq:
    var text = str(clipboard())

    if (text) {
      var newText = text.replace(/\\r\\n/g,'\\r\\n\\r\\n')
      copy(newText)  
      paste()
    }"
IsGlobalShortcut=true
Icon=\xf303
GlobalShortcut=meta+shift+r
```

How to fix this?

**Comment by ramisedhom** [comment, marked as answer]
Got it, here is the script:
```
[Command]
Name=Add extra line in selection
Command="
    copyq:
    copy()
    var text = str(clipboard())
    const regex = /(?:\\r\\n|\\r|\\n)/g;

    if (text) {
      var newText = text.replace(regex,\"\\r\\n\\r\\n\")
      copy(newText)  
      paste()
    }"
IsGlobalShortcut=true
Icon=\xf303
GlobalShortcut=meta+shift+r
```

**Comment by dbareis** [comment]
So you have a workaround too a bug, let's hope the big gets fixed.

If I had to corrupt my coffee that way if be using source() to include my
code and that is the only command I use in all my commands.  It makes
everything (except export, import) so much easier anyway.
